website/profile/views.py

```diff
@@ -87,6 +87,9 @@ def update_user(auth):
     # TODO: Expand this to support other user attributes
     if 'timezone' in data:
         user.timezone = data['timezone']
+    if 'locale' in data:
```

Comment:
Good catch. I can also handle the case where `data['timezone']` is `None` or an empty string.